doc/source/projections.rst

```diff
@@ -0,0 +1,71 @@
+.. currentmodule:: geopandas
+
+.. ipython:: python
+   :suppress:
+
+   import geopandas as gpd
+   world = gpd.GeoDataFrame().from_file('source/_example_data/naturalearth_lowres.shp')
+
+
+
+Managing Projections
+=========================================
+
+
+
+Coordinate Reference Systems
+-----------------------------
+
+CRS are important because the geometric shapes in a GeoSeries or GeoDataFrame object are simply a collection of coordinates in an arbitrary space. A CRS tells Python how those coordinates related to places on the Earth.
```

Comment:
Yeah, struggled with that one. I figured looked better without the "s" but am happy to add.